src/note-components/Homepage/homepage.jsx

```diff
@@ -13,37 +54,40 @@ return (
     <div className="right-section">
       <div className="note-con">
         <div className="note-header">
-          <textarea type="text" placeholder="Start Writing your note ..." className="note-area text-color"></textarea>
+          <input type="text" name="search" placeholder="Title of the note ..." value={notes.title}
+            className="inp-title text-color" onChange={(e)=> setNote(() => ({ ...notes, title: e.target.value }))}/>
           <span><i class="far fa-thumbtack"></i></span>
         </div>
+        <textarea type="text" placeholder="Start Writing your note ..." value={notes.mainContent}
+          className="note-area text-color"
+          onChange={(e)=> setNote(() => ({...notes, mainContent: e.target.value}))}></textarea>
         <div className="note-footer">
-          <label className="footer-text text-color">Created on </label>
-          <input type="date" className="date" />
           <div className="footer-icons">
+            <button onClick={createNoteFunction} className="icon-no-bg"><i class="far fa-plus"></i></button>
             <span><i class="far fa-palette"></i></span>
             <span><i class="far fa-tag"></i></span>
             <span><i class="far fa-archive"></i></span>
```

Comment:
Can be made into one component and just pass the classname as props. As you are repeating this format for of span and i tag at multiple places.